### PR TITLE
rpc: add `getstateroot` to RPC client

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Turns out we're missing it from RPC client. Thanks to @carpawell.

### CODE REVIEW

File: pkg/rpc/client/rpc.go [hunk 1]
```diff
@@ -459,7 +459,7 @@ func (c *Client) GetState(stateroot util.Uint256, historicalContractHash util.Ui
 func (c *Client) FindStates(stateroot util.Uint256, historicalContractHash util.Uint160, historicalPrefix []byte,
 	start []byte, maxCount *int) (result.FindStates, error) {
 	var (
-		params = request.NewRawParams(stateroot.StringLE(), historicalContractHash.StringLE(), historicalPrefix, historicalPrefix, start)
+		params = request.NewRawParams(stateroot.StringLE(), historicalContractHash.StringLE(), historicalPrefix, start)
```
Comment (roman-khimov) [inline pkg/rpc/client/rpc.go:462]: Wow.

Comment (roman-khimov) [reply pkg/rpc/client/rpc.go:462]: But can we make `start` optional? It is in the server implementation. Maybe treat `start == nil` as missing `start`?

Comment (AnnaShaleva) [reply pkg/rpc/client/rpc.go:462]: Yes, we can, but then this behaviour will differ from the C# node. C# node throws an exception if `start == nil` (valid way to pass an empty start is to use `[]byte{}`).

Support nil start?

Comment (AnnaShaleva) [reply pkg/rpc/client/rpc.go]: I added several checks to the client side, so now user will get the right answer even with nil `prefix` or `start`. But we still may extend our server handler to accept nil parameters.

Comment (roman-khimov) [reply pkg/rpc/client/rpc.go]: I think it's OK to handle `nil` this way, it will work server-side and the intention is quite clear.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**